Cloud release notes july 2024 (#6347)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/overview-2024.md

# Overview 2024

### [July 2024](2024-07-releasenotes.md)

### Key Takeaways

> * **Sustainability Dashboard** - The Sustainability Dashboard is available now to all Umbraco Cloud users. This tool is part of our ongoing commitment to supporting greener, more sustainable cloud solutions. We’re excited to see how it helps you in your journey towards a more eco-friendly digital presence

Reviewer: ```suggestion
* **Sustainability Dashboard** - The Sustainability Dashboard is available now to all Umbraco Cloud users. This tool is part of our ongoing commitment to supporting greener, more sustainable cloud solutions. We are excited to see how it helps you on your journey towards a more eco-friendly digital presence.
```
